packages/utils/cli/src/scan/utils/logger.mjs

```diff
@@ -1,19 +1,50 @@
 import chalk from 'chalk'
 
-export const logger = {
+export class Logger {
```

Comment:
Why not using `packages/utils/cli/src/core/Logger.mjs` directly? It seems we duplicate things :thinking: